packages/web3-hooks/base/src/useGasLimitRange.ts

```diff
@@ -2,9 +2,12 @@ import { EMPTY_LIST, type NetworkPluginID } from '@masknet/shared-base'
 import type { HubOptions } from '@masknet/web3-providers/types'
 import { useChainContext } from './useContext.js'
 import { useWeb3Hub } from './useWeb3Hub.js'
+import { useDebugValue } from 'react'
 
 export function useGasLimitRange<T extends NetworkPluginID = NetworkPluginID>(pluginID?: T, options?: HubOptions<T>) {
     const { chainId } = useChainContext<T>({ chainId: options?.chainId })
     const Hub = useWeb3Hub(pluginID, options)
-    return Hub.getGasLimit?.(chainId) || EMPTY_LIST
+    const value = Hub.getGasLimit?.(chainId) || EMPTY_LIST
+    useDebugValue(value)
+    return value
```

Comment:
```suggestion
    return Hub.getGasLimit?.(chainId) || EMPTY_LIST

```

Reply:
we should keep those. it will be easier to debug in the react devtools